code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -15,15 +17,27 @@
 							PRUDENCE_ATTRIBUTE = 60
 							)
 
+/obj/item/ego_weapon/lamp/attack(mob/living/M, mob/living/user)
+	..()
+	for(var/mob/living/L in livinginrange(1, M))
```

Comment:
Doesn't hit SHRIMP!

Reply:
Hey there! I just tested this on the live server. 
It does work. Maybe It wasn't TMed at the time?